css/style.css

```diff
@@ -350,10 +406,182 @@ textarea:focus {
 }
 
 @media (min-width: 768px) {
+  /* Headline */
+  #menu-icon {
+    display: none;
+  }
+
+  #menu {
+    display: block;
+    flex-direction: row;
+    gap: 10px;
+    position: absolute;
+    right: 70px;
+    top: 20px;
+  }
+
+  #menu a {
+    width: 86px;
+    height: 40px;
+    padding: 10px 12px 10px 12px;
+    gap: 10px;
+    text-decoration: none;
+    color: #344563;
+    font-size: 15px;
+    line-height: 20px;
+    text-align: center;
+  }
+
+  #menu a:hover {
+    border: 1px solid gray;
+  }
+
+  #headline {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    background-image: url("../images/Header\ \ desktop\ bg.png");
+    background-repeat: no-repeat;
+    background-size: 100% 90%;
+    border-bottom-left-radius: 90px;
+    background-color: white;
+  }
```

Comment:
- [ ] You have done great job so far, :clap: , However, as part of the project requirement that says: `You must replicate the design as much as possible (e.g., font, colors, images, text, margins)` using this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&mode=design&t=g3dMloiCFcFPsC5J-0). matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Figma</td>
    </tr>
    <tr>
      <td>

 
![FigStudentHome](https://github.com/shimels1/portfolio_microverse/assets/43528594/8a6ec3db-6b52-4e9e-a309-da9246f224f2)

 
</td>
      <td>

![StudentHome](https://github.com/shimels1/portfolio_microverse/assets/43528594/2f9d5c45-02b4-4fa2-af00-2755c130670f)

</td>
    </tr>
  </table>